web/annotation_layer_builder.css

```diff
@@ -77,6 +77,22 @@
   border: 1px solid transparent;
 }
 
+.annotationLayer .textWidgetAnnotation input.comb {
```

Comment:
Don't you want to add these rules to the corresponding file in `/test` as well, to enable testing of `comb` fields once we have better tests?